taskcluster/mozillavpn_taskgraph/transforms/signing.py

```diff
@@ -13,12 +13,17 @@
 transforms = TransformSequence()
 
 PRODUCTION_SIGNING_BUILD_TYPES = [
-    "android-nightly",
-    "android-release"
+    "android-x64/release",
+    "android-x86/release",
+    "android-arm64/release",
+    "android-armv7/release"
 ]
 
 SIGNING_BUILD_TYPES = PRODUCTION_SIGNING_BUILD_TYPES + [
-    "android-debug",
+# Note: it appears we don't have infra for debug sign
+# contact releng if you need it :) 
+#    "android-debug",
+#
```

Comment:
Yeah, we only found one set of credentials to use. I think if we want to sign debug things we'll either need to re-use some keys from another project, or ask SecOps to generate a new set for us. I'll add a JIRA ticket to our board to track it (but likely won't get prioritized until someone pushes for it).

Reply:
No need to take action for this - i just left it as a breadcrumb in case someone later wants to add it. I don't think we really have a use-case for debug signing rn :D